Library/Formula/icetouch.rb

```diff
@@ -0,0 +1,22 @@
+class Icetouch < Formula
+  homepage "https://zeroc.com"
+  url "https://github.com/zeroc-ice/icetouch.git",
+    :tag => "v3.6.0", :revision => "f443edded03fac7f4e5c353ebc87efaf4bfa5229"
+
+  depends_on "mcpp"
+
+  def install
+    # Unset ICE_HOME as it interferes with the build
+    ENV.delete("ICE_HOME")
+    ENV.delete("USE_BIN_DIST")
+    ENV.delete("CPPFLAGS")
+    ENV.O2
+
+    args = %W[
```

Comment:
If you're not modifying the argument array, you can just pass these as arguments directly to `make`.